app/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder.rb

```diff
@@ -43,47 +41,37 @@ def tag
     "<rubocop />"
   end
 
-  def build_messages
-    write_header
-    files.empty? ? write_success : write_offenses
-    @messages.collect!(&:string)
-  end
-
-  def write(line)
-    if message.length + line.length + 1 >= GITHUB_COMMENT_BODY_MAX_SIZE
-      @message = StringIO.new
-      @messages << message
-      write_header_continued
-    end
-
-    message.puts(line)
-  end
-
-  def write_header
+  def header
     commit_range = [
       branch.commit_uri_to(commits.first),
       branch.commit_uri_to(commits.last),
     ].uniq.join(" .. ")
-    write("#{tag}Checked #{"commit".pluralize(commits.length)} #{commit_range} with rubocop #{rubocop_version}")
+    header1 = "Checked #{"commit".pluralize(commits.length)} #{commit_range} with rubocop #{rubocop_version}"
 
     file_count    = results.fetch_path("summary", "target_file_count").to_i
     offense_count = results.fetch_path("summary", "offense_count").to_i
-    write("#{file_count} #{"file".pluralize(file_count)} checked, #{offense_count} #{"offense".pluralize(offense_count)} detected")
+    header2 = "#{file_count} #{"file".pluralize(file_count)} checked, #{offense_count} #{"offense".pluralize(offense_count)} detected"
+
+    "#{tag}#{header1}\n#{header2}"
```

Comment:
Yeah I guess?  The header more matters for the continuation records since it's repeated, and I thought it would make the code more consistent to have a header on every message.  Note that concept didn't really change here...the method was originally called write_header.